src/ProcessMaker/Nayra/Contracts/Bpmn/CatchEventInterface.php

```diff
@@ -23,4 +25,6 @@ public function getEventDefinitions();
      * @return \ProcessMaker\Nayra\Engine\ExecutionInstance[]
      */
     public function getTargetInstances(EventDefinitionInterface $message, TokenInterface $token);
+
+    public function registerCatchEvents(EngineInterface $engine);
```

Comment:
Add documentation to  this method signature

Reply:
Completed